src/most.c

```diff
@@ -531,7 +531,10 @@ int most (int argc, char **argv)
    Most_Program = argv[0];
    piped = 0;
 
-   switches = getenv ("MOST_PROMPT");
+   if (strcmp(getenv("MOST_SECURE"), "1") == 0 || strcmp(getenv("PAGERSECURE"), "1") == 0)
```

Comment:
If `MOST_SECURE` or `PAGERSECURE` are not set getenv will return NULL and passing NULL to strcmp is undefined behavior.